### ISSUE TITLE
data augmentation

### ISSUE THREAD

**Issue by guuusss** [NVIDIA/flownet2-pytorch#120]
Data augmentation on Flownet paper are listed as follows

1. Specifically we sample translation from a the range [−20%; 20%] of the image width for x and y; 

2. rotation from[−17◦; 17◦]; scaling from [0:9; 2:0]. 

3. The Gaussian noise has a sigma uniformly sampled from [0; 0:04]; 

4. contrast is sampled within [−0:8; 0:4]; 

5. multiplicative color changes to the RGB channels per image from [0:5; 2]; 

6. gamma values from [0:7; 1:5] and additive brightness changes using Gaussian with a sigma of 0:2.

Do you mean every training set should use these above?When i test these on tensorflow,i found epe on  validation set go down very slowly.
